### PR TITLE
[README] Breaf introduction to interaction nets.

### PR DESCRIPTION
**Pull request by gsvgit**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,3 +28,54 @@ We use [Fourmolu](https://fourmolu.github.io/) as a formatter for Haskell source
...
+
+then we can replace $a \bowtie b$ with $\mathcal{m}$ and get new network $\mathcal{n_1}$.
+Thus, computation is a sequence of steps of the form $\mathcal{n_i} \xrightarrow{r} \mathcal{n_{i+1}}$.
+Computation finishes when network in normal form.
+
+Active pair $a \bowtie b$ is **realizable** if there is a sequence
+
+$\mathcal{n_0} \xrightarrow{r_0} \ldots \xrightarrow{r_{k-1}} \mathcal{n_k} \ \ $
+
+, $r_i \in \mathcal{R}$, such that $\mathcal{n}_k$ contains $a \bowtie b$.
```
Comment (WoWaster) [inline README.md:80]:
```suggestion
$\mathcal{n_0} \xrightarrow{r_0} \ldots \xrightarrow{r_{k-1}} \mathcal{n_k} \ \ $,

$r_i \in \mathcal{R}$, such that $\mathcal{n}_k$ contains $a \bowtie b$.
```

**Review by WoWaster — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (gsvgit) [reply README.md:80]: @WoWaster Unfortunately, it looks ugly. Thanks to GitHub rendering.

Screenshot from 2024-10-02 07-47-36

Comment (WoWaster) [reply README.md:80]: Ok, let it be this way, then

Comment (KubEF) [inline README.md:80]: Is this not how it should be?
```suggestion
, $r_i \in \mathcal{R}$, such that $\mathcal{n}_i$ contains $a \bowtie b$.
```

Comment (WoWaster) [reply README.md:80]: Block math can fix this issue.
```md
$$\mathcal{n_0} \xrightarrow{r_0} \ldots \xrightarrow{r_{k-1}} \mathcal{n_k},$$
```
look fine to me
image

Comment (gsvgit) [reply README.md:80]: @KubEF I mean the there is a sequence of rewriting that leads to network with the pair. So,  $\mathcal{n}_k$ should contains $a \bowtie b$.

File: README.md [hunk 2]
```diff
@@ -28,3 +28,54 @@ We use [Fourmolu](https://fourmolu.github.io/) as a formatter for Haskell source
...
+
+then we can replace $a \bowtie b$ with $\mathcal{m}$ and get new network $\mathcal{n_1}$.
+Thus, computation is a sequence of steps of the form $\mathcal{n_i} \xrightarrow{r} \mathcal{n_{i+1}}$.
+Computation finishes when network in normal form.
+
+Active pair $a \bowtie b$ is **realizable** if there is a sequence
+
+$$\mathcal{n_0} \xrightarrow{r_0} \ldots \xrightarrow{r_{k-1}} \mathcal{n_k},$$
+
+, $r_i \in \mathcal{R}$, such that $\mathcal{n}_k$ contains $a \bowtie b$.
```
Comment (WoWaster) [inline README.md:80]:
```suggestion
$r_i \in \mathcal{R}$, such that $\mathcal{n}_k$ contains $a \bowtie b$.
```

**Review by KubEF — APPROVED** [review]

**Pull request merged**